packages/next/package.json

```diff
@@ -112,5 +111,8 @@
   },
   "funding": [
     "https://trpc.io/sponsor"
-  ]
+  ],
+  "dependencies": {
+    "@noble/hashes": "^1.3.3"
```

Comment:
Use builtin crypto module?

Reply:
this code will only ever be run serverside, no?

and the crypto module is implemented in all major players including node, bun, workers, edge etc etc

Reply:
I'm not sure to be honest, i'm not that deep in the codebase, but I trust you on that.

Will change it to use crypto module instead!